server/src/main/java/org/elasticsearch/ingest/IngestDocument.java

```diff
@@ -957,6 +958,8 @@ void resetTerminate() {
         terminate = false;
     }
 
+    // Unconditionally deprecate the _type field once V7 BWC support is removed
+    @UpdateForV9(owner = UpdateForV9.Owner.DATA_MANAGEMENT)
```

Comment:
I moved this from another area in order to keep the message/reminder.  I changed it from v10 to v9, as I think we only need BWC to 8 now, and this only exists in v7 IIUC: https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html

Reply:
As mentioned in https://github.com/elastic/elasticsearch/pull/116259, this should be v10 due to a bug where it was never actually deprecated.